RELEASE_NOTES.md

```diff
@@ -1,5 +1,31 @@
 # Release Notes
 
+## 1.7.1
+### Features
+- Rootless Podman now uses Rootlesskit for port forwarding, which should greatly improve performance and capabilities
```

Comment:
s/RootlessKit/Rootlesskit/  Might be good to add a github link to it too '[RootlesKit](https://github.com/rootless-containers/rootlesskit)'